app/models/host/discovered.rb

```diff
@@ -146,9 +146,11 @@ def refresh_facts
     raise ::Foreman::WrappedException.new(e, N_("Could not get facts from proxy %{url}: %{error}"), :url => proxy_url, :error => e)
   end
 
-  def reboot(node_ip = nil)
+  def reboot(old_ip = nil, new_ip = nil)
```

Comment:
```suggestion
  def reboot(old_ip: nil, new_ip: nil)
```

Might make the API a little bit better to read, WDYT?